tutorials/3.Interactions.md

```diff
@@ -0,0 +1,348 @@
+# Making and managing Interactions
+
+A recent change in Discord added some pretty neat ways of interacting. Unfortunately, it requires a fair amount of fiddling, so let's go through how to use this!
```

Comment:
Maybe add a screenshot here showing what these interactions are, and what the user can expect to get out of this tutorial?